src/components/demo-doc.tsx

```diff
@@ -0,0 +1,19 @@
+import { WrappedDocument } from "@govtechsg/open-attestation";
+import React from "react";
+import ROPSTEN_DEMO from "./wrapped/wrapped-demo.json";
+
+interface DemoDocProps {
+  setRawDocument: (doc: WrappedDocument) => void;
+}
+
+export const DemoDoc: React.FunctionComponent<DemoDocProps> = ({ setRawDocument }: DemoDocProps) => {
```

Comment:
it's not used ?

Reply:
DemoDoc is being used in home.tsx

Reply:
https://deploy-preview-15--verify-gov-sg.netlify.app/

where ?

Reply:
![image](https://user-images.githubusercontent.com/503185/83601400-1cd20680-a5a3-11ea-922d-4ba73d4cf170.png)

Reply:
can you try to clear cache and deploy on netlify? because it is running fine on local but now showing up on netlify

Reply:
I fixed this issue